frontend/src/app/shared/community-agreement/community-agreement.widget.css

```diff
@@ -0,0 +1,11 @@
+.community-agreement-dialog {
```

Comment:
NIT: Not necessary, but if you want to, you could bring over the scrollbar styling from the Event Details Card! The default scrollbar is just a little clunky compared to the style of the XL site.

Reply:
Done!